packages/twenty-server/src/metadata/relation-metadata/relation-metadata.service.ts

```diff
@@ -202,6 +202,7 @@ export class RelationMetadataService extends TypeOrmQueryService<RelationMetadat
               action: WorkspaceMigrationColumnActionType.CREATE,
               columnName: foreignKeyColumnName,
               columnType: 'uuid',
+              isNullable: true,
```

Comment:
@Weiko Should be based on the nullability of the relation

Reply:
@magrinj yes I can add isNullable true on the fields created above but since we are creating them I think they will always be nullable (until we provide an API so specify this for the user)

Reply:
We should definitely improve this part in the future and use generator utils like you did with fieldmetadata service 👍 @magrinj